explorer/app/config/model.ts

```diff
@@ -25,6 +26,16 @@ export interface EntityConfig<
   staticLoad?: boolean;
 }
 
+export interface ComponentObject<
```

Comment:
Hi Bruce, can you rename this to be `ComponentConfig`?

Reply:
Good call!